src/components/Attributes/AttributeRow.tsx

```diff
@@ -160,6 +162,7 @@ const AttributeRow: React.FC<AttributeRowProps> = ({
       return (
         <BasicAttributeRow label={attribute.label}>
           <RichTextEditor
+            key={entityId} // temporary workaround, TODO: refactor rich text editor
```

Comment:
I don't see any usage of `key` in `RichTextEditor` component and `key` is also reserved prop for React lists, so how does it solve the problem?

Reply:
It's an ugly workaround that causes react to rerender the component (and lose part of its state like scroll position).